Source/Core/Common/TraversalClient.h

```diff
@@ -0,0 +1,81 @@
+// This file is public domain, in case it's useful to anyone. -comex
+
+#pragma once
+#include <functional>
+#include <list>
+#include <memory>
+#include "Common/Common.h"
+#include "Common/Thread.h"
+#include "Common/TraversalProto.h"
+#include "enet/include/enet/enet.h"
```

Comment:
Since it's an external it should be referred to by `<` and `>` instead of quotations.